### DISCUSSION TITLE
Welcome to dipstick Discussions!

### DISCUSSION THREAD

**Discussion by fralalonde** [fralalonde/dipstick#83] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by priyankat99** [comment]
is there a way to use dipstick to get percentiles for metrics (aggregating metrics so i can the 25th, 50th, and 75th percentile?)

**Comment by fralalonde** [comment]
No, sorry. Dipstick only tracks general statistics that can be maintained in a fixed amount of memory / computation. Determining percentiles (or histograms) would either require:

- keeping track of every point recorded during a period of time

or

- having baked-in percentiles numbers, none of which would satisfy every user (some want 90th, 95th or 99th, or...)

So the idea is to use Dipstick to pump metrics into a dedicated timeseries database and do the fancy metrics there.

See [non-goals](https://github.com/fralalonde/dipstick#non-goals)

**Reply by priyankat99** [reply]
thank you for the reply, i was exploring using the custom publish to try to aggregate metrics for percentiles but even that would not be possible, correct?

**Reply by fralalonde** [reply]
Using custom publish anything should be possible as it is going "out" of dipstick's core. I'd just plug data into Graphite or Influx at that point but you're obviously free to do whatever.

**Comment by fralalonde** [comment]
You may also want to have a look at [this table](https://github.com/fralalonde/dipstick/blob/master/HANDBOOK.md#bucket) to know what metrics are tracked.
